bin/alluxio-start.sh

```diff
@@ -106,12 +106,9 @@ check_mount_mode() {
       fi
       is_ram_folder_mounted "${tier_path}"
       if [[ $? -ne 0 ]]; then
-        if [[ $(uname -s) == Darwin ]]; then
-          # Assuming Mac OS X
-          echo "ERROR: NoMount is not supported on Mac OS X." >&2
-          echo -e "${USAGE}" >&2
-          exit 1
-        fi
+        echo "ERROR: Ramdisk" ${tier_path} "is not mounted with mount option NoMount. Use alluxio-mount.sh to mount ramdisk." >&2
```

Comment:
can we use just one pair of quotes around the whole message? I think " ${tier_path} " should still expand the value of tier_path